content/swiftui/concepts/views/terms/color/color.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'Color'
+Description: 'Creates a color view, modifying the background color.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **Color** view is a way to incorporate colors in the user interface. It can be used to modify the backgrounds.
+
+## Syntax
+
+```pseudo
+var body: some View {
+    Color.color-example
+    //Content that is being modified
+        .foregroundColor(.color-example) //Modifier
+}
+```
+- `Color.color-example`  changes the background color.
+- .foregroundColor(.color-example) changes the color of the content (ex: "text") itself.
+
+## Example
+The following example changes the colors to white on black:
+
+```swift
+var body: some View {
```

Comment:
There is a lot of extra spacing in this code example, can we change this to remove the extraneous spacing? In Codecademy Docs entries, we typically use 2-spaces for indentation in code examples.